public/app/components/fontViewer/fontViewerTemplate.html

```diff
@@ -0,0 +1,32 @@
+<!--<input type="range" id="myRange" min="30" max="150" value="fontViewerCtrl.fontSize" ng-model="fontViewerCtrl.fontSize"-->
+       <!--ng-change="fontViewerCtrl.redrawFont()">-->
+
+<h3 class="font-view-header" id="font-name"></h3>
+<div class="main-section">
+  <div class="glyph-view-container">
+    <div class="glyph-description">
+      <span class="info" id="selected-glyph-name"></span>
+      <span class="info unicode" id="selected-glyph-unicode"></span>
+    </div>
+    <div id="glyph-display" class="glyph-display">
+      <canvas class="glyph-bg" id="glyph-bg" width="250" height="250"></canvas>
+      <canvas class="glyph" id="glyph" width="250" height="250"></canvas>
+    </div>
+    <div class="glyph-data">
+      <span class="info" id="selected-glyph-width"></span>
+      <span class="info index" id="selected-glyph-index"></span>
+    </div>
+  </div>
+
+  <div class="glyphs-table-container">
+    Glyphs
+    <div class="glyphs-table">
+      <div id="glyph-list-end" class="glyph-list-end"></div>
+    </div>
+    Page <span class="pagination"></span>
+  </div>
+</div>
+<div class="footer">
+  <canvas id="preview-canvas" height="40"></canvas>
```

Comment:
trying doing it like this instead:
http://codepen.io/neilkalman/pen/jAxBJW

it gives the canvas a nice clean cut + makes it scroll if needed.